docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,269 @@
+
+```yaml
+remote_write:
+  - url: http://<ingress-host>/api/v1/push
+```
+
+The configuration for an agent that scrapes itself for metrics and writes those metrics to Grafana Mimir looks similar to this:
+
+```yaml
+server:
+  http_listen_port: 12345
+  grpc_listen_port: 54321
+
+metrics:
+  wal_directory: /tmp/grafana-agent/wal
+
+  configs:
+    - name: agent
+      scrape_configs:
+        - job_name: agent
+          static_configs:
+            - targets: ["127.0.0.1:12345"]
+      remote_write:
+        - url: http://<ingress-host>/api/v1/push
+```
+
+Assuming this configuration is written to a `config.yaml` and there is a directory for writing the write ahead log (WAL), the following command can quickly start a Grafana Agent instance:
+
+```bash
+docker run --network=host  -v <path-to-wal-directory>:/etc/agent/data -v <path-to>/config.yaml:/etc/agent/agent.yaml grafana/agent:latest
+```
+
+### Query data in Grafana
+
+First install Grafana, and then add Mimir as a Prometheus data source.
+
+#### Install Grafana
+
+You can either [deploy Grafana Mimir on Kubernetes](https://grafana.com/docs/grafana/latest/setup-grafana/installation/kubernetes/)
+or get a test instance of a local Grafana server up and running
+quickly by using Docker:
+
+```bash
+docker run --rm --name=grafana --network=host grafana/grafana
+```
+
+#### Add Grafana Mimir as a Prometheus data source
+
+1. In a browser, go to the Grafana server at [http://localhost:3000/datasources](http://localhost:3000/datasources).
+1. Sign in using the default username `admin` and password `admin`.
+1. Configure a new Prometheus data source to query the local Grafana Mimir server using the following settings:
+
+   | Field | Value                              |
+   | ----- | ---------------------------------- |
+   | Name  | Mimir                              |
+   | URL   | http://\<ingress-host\>/prometheus |
+
+To add a data source, refer to [Add a data source](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/).
+
+### Verify success
+
+When you have completed the tasks in this getting started guide, you can query metrics in [Grafana Explore](https://grafana.com/docs/grafana/latest/explore/)
+as well as create dashboard panels using the newly configured Grafana Mimir data source.
+
+## Accessing Grafana Mimir from inside the Kubernetes cluster
+
+This chapter does not assume that an ingress is set up, but of course it does not prohibit it either. You can mix the two approaches as needed.
+
+### Enable Meta Monitoring in Grafana Mimir
+
+Grafana Mimir meta monitoring collects metrics and or logs about Grafana Mimir itself. It's primary purpose is to send meta monitoring information to some external receiver, for example a [free tier Grafana Metrics account](https://grafana.com/metrics/). In this example we'll use it to collect and send metrics into itself for testing.
```

Comment:
@dimitarvdimitrov  I've written some words here about meta monitoring, but I'm pretty sure you have something better for a short intro on meta monitoring :)